app/services/gogovan.rb

```diff
@@ -63,10 +64,8 @@ def get_pickup_date
 
   def ggv_driver_notes
     delivery = offer.delivery
-    if offer && delivery
-      user = offer.created_by
-      id = "#{offer.id}-$$-#{user.first_name}-$$-#{user.last_name}-$$-#{delivery.id}"
-      link = "#{Rails.application.secrets.base_urls["app"]}/ggv_order/#{id}"
+    if offer && ggv_uuid
+      link = "#{Rails.application.secrets.base_urls["app"]}/ggv_order/#{ggv_uuid}"
```

Comment:
Line is too long. [82/80]<br>Prefer single-quoted strings inside interpolations.